odoo/addons/base/tests/test_ir_attachment.py

```diff
@@ -260,6 +260,13 @@ def test_13_rollback(self):
         self.Attachment._gc_file_store_unsafe()
         self.assertFalse(os.path.isfile(store_path), 'file removed')
 
+    def test_14_invalid_mimetype_with_correct_file_extension_no_post_processing(self):
+        # test with fake svg with png mimetype
+        unique_blob = b'<blob1>'
```

Comment:
Maybe close the tag ? And pretend a bit more this is an SVG ? (This could be any XML) `<svg xmlns="http://www.w3.org/2000/svg"></svg>` ?

The commit title is very vague and does not say what this does, so instead of `[FIX] base: Check img.image for null`, what about something like `[FIX] base: avoid failing when uploading image with wrong extension` ?